src/components/anti-everyone.ts

```diff
@@ -0,0 +1,47 @@
+import * as Discord from "discord.js";
+import { Message } from "discord.js";
+import { BotComponent } from "../bot-component.js";
+import { Wheatley } from "../wheatley.js";
+import { colors } from "../common.js";
+
+const EVERYONE_EMBED = new Discord.EmbedBuilder()
```

Comment:
I'd prefer ditching the global variable and just constructing the embed below since the variable is only used in one place.

Reply:
Good point. I was structuring this code after `anti-screenshot.ts`, which does the same thing. I will move it inline.

Reply:
Oh yes it is, I'll fix that :)